glide.yaml

```diff
@@ -26,6 +26,9 @@ import:
   subpackages:
   - assert
   - require
+- package: github.com/golang/mock
```

Comment:
According to the convo with @abhinav this shouldn't be necessary. Please convince me it is. I don't like this.